src/db/delivery_problems/mod.rs

```diff
@@ -202,7 +207,10 @@ impl From<LegacyDeliveryProblem> for DeliveryProblem {
             address: source.address,
             problem_type: source.problem_type,
             problem_subtype: source.problem_subtype,
-            created_at: source.created_at,
+            created_at: Utc.timestamp(
```

Comment:
there is a https://docs.rs/chrono/0.4.6/chrono/offset/trait.TimeZone.html#method.timestamp_opt as well, but hopefully we won't need it here for `source.created_at`

Reply:
Oh right, yeah, didn't notice that this one panics. I'll switch.